README.md

```diff
@@ -20,6 +20,8 @@ Store](https://chrome.google.com/webstore/detail/enhanced-keyboard-navigat/coham
 *   `/`/`Escape`: Focus on input search box
 *   `Enter`/`Space`: Navigate to selected result
 *   `Ctrl+Enter`/`⌘+Enter`/`Ctrl+Space`: Open selected result in new tab/window
+*   `←`: Navigate to previous search result page
+*   `→`: Navigate to next search result page

```

Comment:
I'm leaning towards different default keybindings, maybe `Ctrl+up/down`. The main reason is that I also plan to add keybindings for navigating between horizontal elements like news stories and tweets.
What do you think?

Reply:
I think navigation between search result pages is more common than navigation between horizontal elements (where other keybinding are available anyway). In my opinion, the most used option should be the easiest shortcut, so I would propose to keep this as the default option.

Reply:
Yeah I agree with your reasoning, and it's a tough call :)
OK, keep it as is, and I'll think about the other use case.